docs/api/migration-guides/qiskit-opflow-module.mdx

```diff
@@ -357,22 +357,11 @@ to initialize it.
 <Admonition type="tip">
   Interpreting [`qiskit.opflow.primitive_ops.PrimitiveOp`](../qiskit/qiskit.opflow.primitive_ops.PrimitiveOp) as a factory class:
 
-  ```{eval-rst}
```

Comment:
Tables were converted via the script from https://github.com/Qiskit/documentation/pull/296